perfkitbenchmarker/vm_util.py

```diff
@@ -276,6 +277,57 @@ def RunThreaded(target, thread_params, max_concurrent_threads=200):
         '\n'.join([stacktrace for stacktrace in exceptions]))
 
 
+def _ExecuteProcCall(target_arg_tuple):
+  """Function invoked in another process by RunParallelProcesses.
+
+  Executes a specified function call.
+
+  Args:
+    target_arg_tuple: (target, args, kwargs) tuple containing the function to
+        call and the arguments to pass it.
+
+  Returns:
+    (result, exception) tuple. The first element is the return value from the
+    called function, or None if the function raised an exception. The second
+    element is the raised exception, or None if the function succeeded.
+  """
+  target, args, kwargs = target_arg_tuple
+  try:
+    return target(*args, **kwargs), None
+  except:
+    return None, traceback.format_exc()
+
+
+def RunParallelProcesses(target_arg_tuples, max_concurrency=None):
+  """Executes function calls concurrently in separate processes.
+
+  Args:
+    target_arg_tuples: list of (target, args, kwargs) tuples. Each tuple
+        contains the function to call and the arguments to pass it.
+    max_concurrency: int. The maximum number of concurrent new processes.
```

Comment:
Per core? The docstring already uses the term "process" - do you want a comment in the docstring comparing it to RunThreaded?
I use this method to parallelize memory-intensive post-processing of data collected in a benchmark I'm working on.